ui/client/components/toolbars/creator/ComponentIcon.tsx

```diff
@@ -13,12 +13,20 @@ export const preloadImage = memoize((href: string) => new Promise<string>(resolv
   return image.onload = () => resolve(href)
 }))
 
-export const getComponentIconSrc = memoize((node: NodeType, processDefinitionData: ProcessDefinitionData) => {
-  if (node) {
-    const nodeComponentId = ProcessUtils.findNodeConfigName(node)
-    const componentConfig = processDefinitionData.componentsConfig?.[nodeComponentId]
+function _getComponentIconSrc(nodeOrPath: string): string
```

Comment:
Why you change it? Using `/api/components` you should use only `absoluteBePath(icon)`. Right now it's to complicated.. :/